scripts/zkai_d128_proof_artifact_backend_spike_gate.py

```
@@ -739,21 +839,43 @@ def validate_payload(payload: Any, *, require_mutations: bool = True) -> None:
-        "parameterized route status",
+        route_by_name["parameterized_vector_residual_add_air"].get("status"),
+        "GO_PARTIAL_D128_RESIDUAL_ADD_ONLY",
+        "parameterized residual-add route status",
+    )
+    expect_equal(
+        route_by_name["parameterized_transformer_block_air"].get("status"),
+        "NO_GO_FULL_BLOCK_SLICES_MISSING",
+        "parameterized full-block route status",
     )
     for route in routes:
         route = require_object(route, "backend route")
```

Ruff (0.15.12)

[warning] `for` loop variable `route` overwritten by assignment target

(PLW2901)